src/main/java/racingcar/domain/Distance.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import java.util.Objects;
+
+public class Distance {
+
```

Comment:
```suggestion

public class Distance implements Comparable<Distance> {

```

Comparable을 구현해보는 것은 어떨까요? 
이번 미션에서 어떻게 사용해볼 수 있을까요?

Reply:
최대값을 찾을 때 사용할거 같은데요.
정렬을 하자니 선형탐색보다 느릴거같고, 우선순위 큐를 활용하는 방법을 말씀하시는걸까요?